src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -203,6 +220,17 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
             return response;
         }
 
+        private static bool GetQueryParameterValue(HttpRequestMessage request, string queryParameterName)
+        {
+            if (!request.GetQueryNameValuePairs().AllKeys.Contains(queryParameterName))
```

Comment:
Rather than checking for the query parameter key and then fetching the value, it's more efficient to check the value always and compare it to null to determine whether it exists. One lookup rather than two.

Reply:
@cgillum  Nice improvement! Thank you! I will modify accordingly.